### ISSUE TITLE
Importing Database makes code analyzer extremely slow

### ISSUE THREAD

**Issue by tomwyr** [JetBrains/Exposed#677]
After importing `org.jetbrains.exposed.sql.Database` into my file the code analyzer slows down to the point where the IDE becomes almost unusable. IntelliJ stops finding code errors, doesn't show import/code hints. Also this happens only in the file where database class is imported. After I remove `Database` class import everything seems to work fine again.

```
IntelliJ IDEA 2019.2.4 (Community Edition)
Build #IC-192.7142.36, built on October 29, 2019
Runtime version: 11.0.4+10-b304.77 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 974M
Cores: 4
Registry: 
Non-Bundled Plugins: org.jetbrains.kotlin
```

**Comment by dzirbel** [comment]
I'm seeing this as well with Exposed 0.17.6. IntelliJ code analysis seems to get stuck (not all the syntax highlighting is there, no code completion, etc), but the IDE is responsive otherwise. Downgrading to Exposed 0.17.5 seems to fix the issue.

`IntelliJ IDEA 2019.2.4 (Community Edition)
Build #IC-192.7142.36, built on October 28, 2019
Runtime version: 11.0.4+10-b304.77 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 2006M
Cores: 4
Registry: ide.tooltip.initialDelay=604
Non-Bundled Plugins: org.jetbrains.kotlin, no.tornado.tornadofx.idea, org.sylfra.idea.plugins.linessorter`

**Comment by tomwyr** [comment]
I can confirm that it was not an issue in version `0.17.5`.

**Comment by Tapac** [comment]
@tomwyr , @dzirbel , could you please report it to Kotlin issue tracker [here](https://youtrack.jetbrains.com/issues/KT) and share issue number?
I can't investigate such problems but will try to ask kotlin team to investigate.

**Comment by tomwyr** [comment]
@Tapac [KT-34759](https://youtrack.jetbrains.com/issue/KT-34759)

**Comment by dzirbel** [comment]
May have been incidental, but this seems to be resolved for me in version 0.17.7.

**Comment by tomwyr** [comment]
This seems to be true. Closing it for now unless there's someone else who still has the issue in the newest version.
